### PR TITLE
added a "sketch" of repo, entity and service

### PR DESCRIPTION
**Pull request by kaiiiseeel**

### CODE REVIEW

File: src/main/java/mate/academy/bookstore/repository/impl/BookRepositoryImpl.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+@Repository
+public class BookRepositoryImpl implements BookRepository {
+    @Override
+    public Book save(Book book) {
+        return null;
+    }
+
+    @Override
+    public List<Book> findAll() {
+        return null;
```
Comment (andrii-hoienko) [inline src/main/java/mate/academy/bookstore/repository/impl/BookRepositoryImpl.java]: Implement logic

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1 +1,6 @@
 spring.application.name=book-store
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
```
Comment (andrii-hoienko) [inline src/main/resources/application.properties]: Don`t push creadentials

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/bookstore/entity/Book.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name = "books")
+public class Book {
```
Comment (karma-o) [inline src/main/java/mate/academy/bookstore/entity/Book.java]: this is not critical for now, but `@Data` is not recommended for use with `@Entity`, details on that:
https://medium.com/@dev.fethi/data-anatation-and-hibernate-n-1-problems-466df982288e

**Review by karma-o — APPROVED** [review]
Left a minor comment and approved

**Pull request merged**